lib/runner/extensions/request.command.js

```diff
@@ -29,43 +30,46 @@ var _ = require('lodash'),
      * @returns {Object}
      */
     createContext = function (payload) {
-        var context = payload.context ? payload.context : _.pick(payload, CONTEXT_PROPERTIES),
+        var context = _.pick(payload, CONTEXT_PROPERTIES),
+            item,
             parent;
 
-        // get a reference to the Auth instance from the original item, so any changes don't have to be synced back
-        context.auth = payload.item.getAuth();
+        // we clone item from the payload, so that we can make any changes we need there, without mutating the 
+        // collection.
+        item = new sdk.Item(payload.item.toJSON());
+
+        // in order to ensure that variable resolution works correctly, we set the __parent property,
+        // so that the SDK can locate the VariableList in the collection.
+        parent = payload.item.parent();
+        parent && (item.setParent(parent));
+
+        // save the cloned item in context
+        context.item = item;
 
-        // if an item is not provided in the context, clone the one from the payload,
-        // so that we can make any changes we need there, without mutating the collection.
-        context.item = context.item || new sdk.Item(payload.item.toJSON());
+        // save original item for reference
+        context.originalItem = item;
```

Comment:
don't you mean payload.item?

Reply:
It should have been `payload.item` but, it won't have `parent` which makes it difficult for replays.